doc/api/net.md

```diff
@@ -343,23 +351,21 @@ added: v0.3.4
 
 Construct a new socket object.
 
-`options` is an object with the following defaults:
-
-```js
-{
-  fd: null,
-  allowHalfOpen: false,
-  readable: false,
-  writable: false
-}
-```
-
-`fd` allows you to specify the existing file descriptor of socket.
-Set `readable` and/or `writable` to `true` to allow reads and/or writes on this
-socket (NOTE: Works only when `fd` is passed).
-About `allowHalfOpen`, refer to [`net.createServer()`][] and [`'end'`][] event.
-
-`net.Socket` instances are [`EventEmitter`][] with the following events:
+* `options` {Object} Available options are:
+  * `fd`: {number | null} Defaults to `null`. If specified, wrap around an
```

Comment:
+1 to describe the meaning before describing default. Also some docs use "Defaults to", some use "Default to". Not sure if we need to use present tense in the docs if a sentence starts this way? (This is a bit inconsistent in almost every API docs I've ever seen, not just Node.js..)

(random idea: it would be great if we have some structural way to describe the default, and let the doc tools generate something with different styles).